gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/statistics/functions/ExtractPropertyValues.java

```diff
@@ -26,8 +26,9 @@
 import java.util.Set;
 
 /**
- * Extracts Triples of the form <Tuple1<PropertyName>, PropertyValue> from the given list of
+ * Extracts Triples of the form {@code <Tuple1<PropertyName>, PropertyValue>} from the given list of
```

Comment:
This does not sound correct. It returns `Tuple2<String,  Set<PropertyValue>>` which are no triples (?)